warehouse/ingest-csv/src/test/java/datawave/ingest/csv/mr/handler/ColumnBasedHandlerTestUtil.java

```diff
@@ -40,9 +41,9 @@
  */
 public class ColumnBasedHandlerTestUtil {
     
-    public static final Text shardTableName = new Text("shard");
-    public static final Text shardIndexTableName = new Text("shardIndex");
-    public static final Text shardReverseIndexTableName = new Text("shardReverseIndex");
+    public static final Text shardTableName = new Text(TableName.SHARD);
+    public static final Text shardIndexTableName = new Text(TableName.SHARD_INDEX);
+    public static final Text shardReverseIndexTableName = new Text(TableName.SHARD_RINDEX);
     public static final Text edgeTableName = new Text("edge");
```

Comment:
No edge table constant?

Reply:
I'll add EDGE as a table name in the next commit.

Reply:
Btw, I created #346 for the cleanup work. If you want to just commit as-is that's fine. Or if you want to go ahead and remove it and the references that you've refactored as part of this PR, that's cool to. Doesn't matter to me

Reply:
This isn't an everything-or-nothing PR, I'm fine leaving that work for a separate issue.